shared/chat/conversation/input/container.js

```diff
@@ -188,7 +189,7 @@ export default compose(
       inputFocus: props => () => input && input.focus(),
       inputBlur: props => () => input && input.blur(),
       inputSelections: props => () => (input && input.selections()) || {},
-      inputSetRef: props => i => {
+      _inputSetRef: props => i => {
```

Comment:
this is obscured here and turns into `inputSetRef` in the mention hoc layer

Reply:
Just curious -- would it be possible to make these handlers part of the HOC itself?

Reply:
I'm not sure if it'd be a good separation of concerns, since the mention HOC just deals with the handlers for user / channel mentions while these are more general input handlers. I am going to work on bringing the native input up to feature parity with the desktop input so some of these might become unnecessary.

Reply:
but I'll bring in the `withState`'s to the hoc, which might be what you meant.